cmd/cluster/logs/cluster.go

```diff
@@ -0,0 +1,111 @@
+package logs
+
+import (
+	"fmt"
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	ocmutils "github.com/openshift/osdctl/pkg/utils"
+	"os"
+)
+
+func fetchDetails(clusterKey string) (string, string, string) {
```

Comment:
**Suggestion:** name parameters or return a struct, it's hard to tell what this function returns from the signature `string, string, string`.